### ISSUE TITLE
 URL Logging direct to txt file

### ISSUE THREAD

**Issue by Digita1-dave** [alct/export-tabs-urls#4]
Please can I request a feature

we used to use  URL Logger by Nor Stenberg(no longer working) that saved the tabs when the user selected them into a text file with like time and date values. We used it to track the url sites users visited, as monitoring on the firewall end gives pages and pages and unnecessary referer urls. When we just want to see what address the users delibratly went on, using browser histroy proved unworkable as it can be deleted.

This text file that just had a single url from the currently select tab in it, we then harvested it when the file mod date changed.

In short add a option to save to a log file the current url of selected tab when it changes

**Comment by alct** [comment]
Hello,

Thank you for your interest in this addon.

> We used it to track the url sites users visited, as monitoring on the firewall end gives pages and pages and unnecessary referer urls. When we just want to see what address the users delibratly went on, using browser histroy proved unworkable as it can be deleted.

I do not want to be involved in any activity related to user tracking.

**Comment by Digita1-dave** [comment]
thats a shame we could have come to a paid agreement,
this is for our company internal staff and is fully within the law an
regulations of the uk.
